11-change-ppp-mtu-debug.sh

```diff
@@ -50,7 +51,10 @@ while true; do
           echo Got $emtu for $einterface
           # Current ethernet MTU is incorrect so needs changing
           echo Checking $einterface
-          if [[ $emtu -lt $(($PTARGET+8)) ]] ; then
+          if [[ $emtu -eq $(($PTARGET)) ]] ; then
```

Comment:
changed this to look for the correct rather than the incorrect but that incorrect is not always right. it would only be right if you were trying to increase the mtu rather than decrease. which is what i want to do. so this works for both